tools/demo/src/main.cpp

```diff
@@ -0,0 +1,51 @@
+#include <Lambda/Lambda.h>
```

Comment:
This file doesn't follow our cpplint style guide which is okay for internal tools, but something to be cautious about in the future and especially so if contributing the lambda source directly.